src/lib/deleteDoc.ts

```diff
@@ -0,0 +1,40 @@
+import type { CommandCategories } from './baseCommand';
+
+import config from 'config';
+import { Headers } from 'node-fetch';
+
+import fetch, { cleanHeaders, handleRes } from './fetch';
+
+/**
+ * Delete a document from ReadMe
+ *
+ * @param {String} key the project API key
+ * @param {String} selectedVersion the project version
+ * @param {Boolean} dryRun boolean indicating dry run mode
+ * @param {String} slug The slug of the document to delete
+ * @param {String} type module within ReadMe to update (e.g. docs, changelogs, etc.)
+ * @returns {Promise<String>} a string containing the result
```

Comment:
Could you move this type to the function definition so TS picks up on it?

Reply:
Done